src/nvim/option_defs.h

```diff
@@ -492,9 +492,10 @@ EXTERN int p_js;                // 'joinspaces'
 EXTERN char_u *p_jop;           // 'jumpooptions'
 EXTERN unsigned jop_flags;
 #ifdef IN_OPTION_C
-static char *(p_jop_values[]) = { "stack", NULL };
+static char *(p_jop_values[]) = { "stack", "view", NULL };
```

Comment:
Need to update the help text in the docs for jumpoptions

Reply:
Let me know what you think https://github.com/muniter/neovim/blob/4897a954bc06b3d6935b53b783285f5994e48f45/runtime/doc/options.txt#L3513